libcst/codemod/visitors/_insert_statements.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# pyre-strict
```

Comment:
@jimmylai Should we flip the repo default to pyre-strict in the pyre config so this isn't needed for new files?

Reply:
Yes, we still have a few files not in strict mode yet. 
We need to fix them first. This is the first step. https://github.com/Instagram/LibCST/pull/271

Reply:
We can flip the default before converting all files, we just need to add `# pyre-unsafe` to the not-fully-strict-yet files.